openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -1562,7 +1442,7 @@ static const char* authentication_method(const SSL* ssl) {
                 // No cipher available so return UNKNOWN.
                 return UNKNOWN_AUTH_METHOD;
             }
-            return cipher_authentication_method(sk_value((_STACK*) ciphers, 0));
+            return SSL_cipher_authentication_method(sk_value((_STACK*) ciphers, 0));
```

Comment:
i would advise against using the `SSL_*` for method name prefix. This namespace is implicitly owned by OpenSSL and  is misleading as to where the method comes from and using the same prefix as another library means the probability of method name conflict goes up.